catlearn/relation_cache.py

```diff
@@ -154,6 +155,13 @@ def data(
         # if arrow has length 0, return data corresponding to its node
         return self._datas[arrow[0]]  # type: ignore
 
+    @property
+    def label_universe(self) -> Mapping[ArrowType, Tsor]:
```

Comment:
Add the same to DecisionCatModel (as it is there is no way to access it from the outside if I'm not mistaken).